k8s/README.md

```diff
@@ -0,0 +1,224 @@
+# Purpose
```

Comment:
There is additional documentation (intended to be more project-level) as part of this PR in main: https://github.com/eclipse-pass/main/pull/282/files